Assets/Resources/data/item_data/items.json

```diff
@@ -0,0 +1,18 @@
+{
+    "items": [
+        {
+            "id": 0,
+            "name":"Basic Health Potion",
+            "description":"Drink to restore 10HP.",
+            "isUseable":true,
+            "isConsumeable":true,
+            "use_effect": 0,
+            "effect_potency":[10],
+            "turns_applied":0,
+            "coin_value": [
```

Comment:
inconsistent naming, unless using pascal for bools? do you have a standard doco somewhere i can read?